quickstarts/karaf-camel-rhosak-template.json

```diff
@@ -0,0 +1,364 @@
+{
+  "apiVersion": "v1",
+  "kind": "Template",
+  "metadata": {
+    "annotations": {
+      "openshift.io/display-name": "Red Hat Fuse 7.9 Camel RHOSAK with Karaf",
+      "openshift.io/provider-display-name": "Red Hat, Inc.",
+      "description": "A simple Camel route for Red Hat Openshift streams for Apache Kafka. This quickstart shows a simple Apache Camel application that connects with Red Hat OpenShift Streams for Apache Kafka.",
+      "tags": "quickstart,java,karaf,fis,jboss-fuse,rhosak",
+      "iconClass": "icon-rh-integration",
+      "version": "1.9"
+    },
+    "name": "s2i-fuse79-karaf-camel-rhosak"
+  },
+  "labels": {
+    "template": "s2i-fuse79-karaf-camel-rhosak"
+  },
+  "parameters": [
+    {
+      "name": "APP_NAME",
+      "displayName": "Application Name",
+      "required": true,
+      "value": "s2i-fuse79-karaf-camel-rhosak",
+      "description": "The name assigned to the application."
+    },
+    {
+      "name": "GIT_REPO",
+      "displayName": "Git Repository URL",
+      "value": "https://github.com/fabric8-quickstarts/karaf-camel-rhosak.git",
+      "required": true,
+      "description": "The URL of the repository with your application source code."
+    },
+    {
+      "name": "GIT_REF",
+      "displayName": "Git Reference",
+      "value": "master",
+      "description": "Set this to a branch name, tag or other ref of your repository if you are not using the default branch."
+    },
+    {
+      "name": "BUILDER_VERSION",
+      "displayName": "Builder version",
+      "value": "1.9",
+      "description": "The version of the FIS S2I builder image to use."
+    },
+    {
+      "name": "APP_VERSION",
+      "displayName": "Application Version",
+      "value": "2.2.60-SNAPSHOT",
```

Comment:
Is it normal having a Snapshot version here?
Why is it 2.2.60, it corresponds to the version of what else?

Reply:
This is actually okay, it gets set within the pipeline to the version of the quickstart that is built (example : https://github.com/jboss-fuse/application-templates/blob/application-templates-2.1.0.fuse-sb2-790042/quickstarts/karaf-camel-log-template.json#L48), but it will be a build time correction